aarch64_extension.cpp

```diff
@@ -23,6 +23,49 @@ template <typename T> inline T Ones(size_t count) {
   }
 }
 
+/*
+ * Given a Vector Arrangement Specifier, return the _element_ size to be
+ * operated on.
+ * Verify that the VAS is indeed valid before calling this function.
+ */
+static size_t vectorElementSize(enum arm64_vas vas) {
+  switch (vas) {
+  case ARM64_VAS_16B:
+  case ARM64_VAS_8B:
+#ifdef CAPSTONE_NEXT
+  case ARM64_VAS_4B:
+  case ARM64_VAS_1B:
+#endif /* #ifdef CAPSTONE_NEXT */
+    return (1);
+  case ARM64_VAS_8H:
+  case ARM64_VAS_4H:
+#ifdef CAPSTONE_NEXT
+  case ARM64_VAS_2H:
+  case ARM64_VAS_1H:
+#endif /* #ifdef CAPSTONE_NEXT */
+    return (2);
+  case ARM64_VAS_4S:
+  case ARM64_VAS_2S:
+#ifdef CAPSTONE_NEXT
+  case ARM64_VAS_1S:
+#endif /* #ifdef CAPSTONE_NEXT */
+    return (4);
+  case ARM64_VAS_2D:
+  case ARM64_VAS_1D:
+    return (8);
+  case ARM64_VAS_1Q:
+    return (16);
+  default:
+    /*
+     * If we've called this with ARM64_VAS_INVALID we're missing new enum
+     * members, we need to fix the caller.
+     */
+    LogError("%s() called with invalid VAS %d", __FUNCTION__, vas);
```

Comment:
Should be a better way to handle errors here, perhaps `std::optional<>`? I'll think about it a little more, but this probably shouldn't be how we handle this.